src/components/Input/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+import { InputCls } from "./styles"
+import { cx } from "emotion"
+
+Input.propTypes = {
+  className: PropTypes.string,
+  onChange: PropTypes.func,
+  label: PropTypes.string,
+  value: PropTypes.string,
+  disabled: PropTypes.bool,
+  errorMessage: PropTypes.string
+}
+
+function Input({ className, onChange, label, value, errorMessage, success, errorForm, disabled, search }) {
+  const [inputValue, setInputValue] = useState("")
+  const [isDisabled, setIsDisabled] = useState(false)
+  const [error, setError] = useState(false)
+
+  errorMessage = "We are sorry, an error occured. Please try again."
+
+  return (
+    <div className={cx(InputCls, { success, errorForm, disabled, search }, className)}>
```

Comment:
errorForm should not be a property of the Input component. It should be assigned only when there is an errorMessage property to the component. eg
```suggestion
    <div className={cx(InputCls, { success, errorForm: Boolean(errorMessage), disabled, search }, className)}>
```

Reply:
![image](https://user-images.githubusercontent.com/54745415/84600762-66b0cb80-ae84-11ea-87ae-c482d10a0bd7.png)

why am i not allowed to commit your suggestion?